src/Carbon/Carbon.php

```diff
@@ -926,13 +926,41 @@ protected static function translator()
         if (static::$translator === null) {
             static::$translator = new Translator('en');
             static::$translator->addLoader('array', new ArrayLoader());
-            static::setLocale('en');
+
+            $locale = self::getDefaultLocale();
+            if ($locale) {
+                static::setLocale($locale);
+            } else {
+                static::setLocale('en');
+            }
         }
 
         return static::$translator;
     }
 
     /**
+     * Gets the locale set in the php configs, converts it to the two letter format.
+     * Returns false if a locale could not be determined.
+     *
+     * Example:
+     * 'en_GB.utf8' to 'en'
+     * 'it_IT.utf8' to 'it'
+     *
+     * @return bool|string
+     */
+    public static function getDefaultLocale()
+    {
+        $defaultLocale = setlocale(LC_TIME, "0");
+        if (strlen($defaultLocale) && $defaultLocale !== 'C') {
```

Comment:
Already pointed that out @ https://github.com/briannesbitt/Carbon/pull/540#discussion_r46583218
But yeah, can also just be removed.